sdk/include/reactos/ts/winsta.h

```diff
@@ -0,0 +1,310 @@
+/*
+ * PROJECT:     Terminal Services (winsta) Private API
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     WinStation private API definitions
+ * COPYRIGHT:   Copyright 2025 Pierce Andjelkovic <pierceandjelkovic@gmail.com>
+ */
+
+#ifndef REACTOS_WINSTA_H
+#define REACTOS_WINSTA_H
+
+// refs: https://github.com/mirror/processhacker/blob/master/2.x/trunk/phlib/include/winsta.h
+//       https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-tsts/ce70794f-2138-43e8-bf6c-2c147887d6a2
+
+/* WinSta calling convention */
+#define WINSTAAPI WINAPI
+//#include <ntdef.h>
+
+#include <stdarg.h>
+
+#include <windef.h>
+#include <winbase.h>
+
+//#include <winnt.h>
+//#include <ntdef.h>
+typedef unsigned char BOOLEAN, *PBOOLEAN;
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#define WINSTATIONNAME_LENGTH 32
+typedef WCHAR WINSTATIONNAME[WINSTATIONNAME_LENGTH + 1];
+
+typedef enum _WINSTATIONSTATECLASS
+{
+    State_Active,
+    State_Connected,
+    State_ConnectQuery,
+    State_Shadow,
+    State_Disconnected,
+    State_Idle,
+    State_Listen,
+    State_Reset,
+    State_Down,
+    State_Init
+} WINSTATIONSTATECLASS;
+
+typedef struct _SESSIONIDW
+{
+    union
+    {
+        ULONG SessionId;
+        ULONG LogonId;
+    } DUMMYUNIONNAME;
+    WINSTATIONNAME WinStationName;
+    WINSTATIONSTATECLASS State;
+} SESSIONIDW, *PSESSIONIDW;
+
+typedef struct _TS_SYS_PROCESS_INFORMATION
+{
+    ULONG NextEntryOffset;
+    ULONG NumberOfThreads;
+    LARGE_INTEGER SpareLi1;
+    LARGE_INTEGER SpareLi2;
+    LARGE_INTEGER SpareLi3;
+    LARGE_INTEGER CreateTime;
+    LARGE_INTEGER UserTime;
+    LARGE_INTEGER KernelTime;
+    VOID* ImageName;
+    //UNICODE_STRING ImageName;
```

Comment:
This needs to be brought back in, but there's ugly dependency issues when it's not commented out